code/drasil-docLang/lib/Drasil/DocumentLanguage/TraceabilityMatrix.hs

```diff
@@ -76,10 +76,9 @@ traceMColumns fc fr c = map ((\u -> filter (`elem` u) $ fc u) . flip traceLookup
 tableShows :: (Referable a, HasShortName a) => a -> Sentence -> Sentence
 tableShows r end = refS r +:+ S "shows the" +:+ plural dependency `S.of_` (end !.)
 
--- | Helper that makes sure the rows and columns of a traceability matrix have substance.
-ensureItems :: String -> [a] -> [a]
-ensureItems u [] = error $ "Expected non-empty matrix dimension for traceability matrix " ++ u
-ensureItems _ l = l
+-- | Helper that prints an error when traceability matrix is empty.
+emptyItems :: UID -> LabelledContent
+emptyItems u = llcc (makeTabRef' u) $ Paragraph $ S $ show u ++ " : is empty/incomplete"
```

Comment:
Some notes about this message (in decreasing importance):
- [ ] The UID displayed here doesn't refer to a specific matrix dimension, but to the whole matrix. I would change this so that it is clear that at least one matrix dimension is empty
   - Maybe it could even determine what set of concepts is empty? This seems unnecessary to me, though.
- [ ] I think this should explicitly say "empty", since an incomplete list will still result in a table being generated correctly.
- [ ] I personally think that there shouldn't be a space before the colon (although others may disagree).

Reply:
I implemented changes and fixed it so it now actually shows the messages.